test/models/playlist_test.rb

```diff
@@ -0,0 +1,23 @@
+require 'test_helper'
+
+class PlaylistTest < ActiveSupport::TestCase
+  test 'valid if danceanbility beetween 1 and 100' do
+    1.upto(100).each do
+      playlist = Playlist.new(danceability: 10)
+      playlist.valid?
+      assert_empty playlist.errors[:danceability]
```

Comment:
Honnêtement tu peux te contenter de vérifier que `valid?` est faux:

```suggestion
      assert_not playlist.valid?
```

Franchement je chipote, mais c'est plus clair de tester que ce n'est pas valide plutôt que la présence d'une erreur sur name (même si in-fine c'est équivalent, et même ton test est plus précis) + en pratique on ne va pas autant dans le détail.